src/commands/misc/pusheen.js

```diff
@@ -12,6 +12,7 @@ module.exports = class Pusheen extends Command {
   async run ({ t, author, channel }) {
     channel.startTyping()
     const getRandom = (r) => r[Math.floor(Math.random() * r.length)]
+    const getRandomPhoto = (r) => r[Math.floor(Math.random() * ( r.length - 1 )) + 1]
```

Comment:
to get a random photo from the photos array that isnt the first one(usually titles)